packages/oruga/src/components/taginput/Taginput.vue

```diff
@@ -223,24 +231,24 @@ const props = defineProps({
 const emits = defineEmits<{
     /**
      * modelValue prop two-way binding
-     * @param value {any[]} updated modelValue prop
+     * @param value {(String | Number | Object)[]} updated modelValue prop
```

Comment:
Just a minor suggestion to use primitive types here, to align with changes within `Autocomplete`, common JS practices and the types expected by TypeScript.

```suggestion
     * @param value {(string | number | object)[]} updated modelValue prop
```